### ISSUE TITLE
Add Calculated Nearest Transit Field

### ISSUE THREAD

**Issue by mzagaja** [MAPC/massbuilds#42]
The current metadata spec calls for adding a calculated nearest transit field. I can transplant this from old mass builds (time permitting) but my recollection is we had a discussion about whatever API we were using was not pulling enough data or something. I remember fixing a bug since this wasn't working right in old MassBuilds and I'm wondering if it might make sense to hold off on this one for a future sprint where we can give it its due. But if we think it works sufficiently, for now, I can try transplanting it over.

**Comment by TimReardon** [comment]
@mzagaja this is a pretty important feature.  You are correct there were some issues with the prior functioning but i think those were resolved. In any case, I think we may be able to approach this differently by providing a shapefile of (non-overlapping) transit station areas and you can determine which one, if any, contains the point.

**Comment by mzagaja** [comment]
I have dug into this one further and learned the MBTA has a new/updated API (https://api-v3.mbta.com/docs/swagger/index.html#/Stop/ApiWeb_StopController_index) that might work better. So I will give that a shot.

**Comment by TimReardon** [comment]
@mzagaja yes but this won't include all the regional transit authority stops outside the MBTA service area. Plus we noted some limitations of using the "closest transit stop" approach, since some stops are more important than others.  I observed some developments that were within a quarter mile of a subway stop being matched to the rinky-dink bus stop at the development's front door, instead of to the more significant subway stop.  We've created a hierarchical set of non-overlapping transit station areas to solve this problem.
